main/get.c

```diff
@@ -829,7 +830,8 @@ extern int cppGetc (void)
 				int next = getcFromInputFile ();
 
 				if (next == NEWLINE)
-					continue;
+					// continue; <-- this may exit returning the backslash, which si wrong.
```

Comment:
Ow...
I could even remove that line... have left it there with a comment to remember what I have changed in case it didn't work, but it seems to work so...